src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -163,6 +163,14 @@ public function note($message)
         $this->block($message, 'NOTE', 'fg=yellow', ' ! ');
     }
 
+    /**
+     * {@inheritdoc}
```

Comment:
there's no "info" method on `StyleInterface`
if we want to add one, this should be done in a BC manner, aka via an `@method` annotation

Reply:
@nicolas-grekas I updated the docblock. I took the same approach that we did for the method _comment_ (it is not part of the interface). What do you reckon?